libtenzir/include/tenzir/flatbuffer.hpp

```diff
@@ -140,9 +140,16 @@ class flatbuffer final {
     if (not result) {
       return std::move(result.error());
     }
+    // FlatBuffers defaults to erroring out after 1M table entries in the
+    // verifier. This was chosen rather randomly and for historic reasons, they
+    // cannot change it. We use the much saner default of not erroring out for
+    // large tables here.
+    auto options = flatbuffers::Verifier::Options{};
+    options.max_tables = std::numeric_limits<flatbuffers::uoffset_t>::max();
     const auto* const data
       = reinterpret_cast<const uint8_t*>(result->chunk()->data());
-    auto verifier = flatbuffers::Verifier{data, result->chunk()->size()};
+    auto verifier
+      = flatbuffers::Verifier{data, result->chunk()->size(), options};
```

Comment:
Thanks for the accompanying explanation!

We also use `flatbuffers::Verifier` for `type` and `table_slice`. For `type` is probably doesn't matter, but perhaps we need the same change for `table_slice`?